dags/gmail_data_import_pipeline.py

```diff
@@ -91,6 +93,10 @@ def get_gmail_user_id():
     return get_env_var_or_use_default(GMAIL_DATA_USER_ID_ENV)
 
 
+def is_gmail_end2end_test():
+    return get_env_var_or_use_default(IS_GMAIL_END2END_TEST_ENV)
```

Comment:
This could potentially cause confusion. e.g. if you set the environment variable to `False`, then it will still think that we are running end-to-end tests. That is because it is just treating it as a string.

Python seem to be missing a simple `to_bool` function. So you could use something like: `value in {`True`}` (not sure if there is already a helper function in the project).

Reply:
I found a way, maybe it can work:
https://docs.python.org/3/distutils/apiref.html#distutils.util.strtobool

Reply:
Or we can create our own function like:

```
def str2bool(value):
  return value.lower() in ("yes", "y", "true", "t", "1")
```

Reply:
I think I have used `strtobool` from `distutils` before.
It is a bit of a shame that it is not part of the more core Python API. i.e. it feels like we shouldn't use it, but it does what we want to do, except that it should return a boolean rather than `1` or `0` (but you might prefer that ;-) ).
The other thing that it does perhaps differently but could be handled is what to do with `None` or blank strings.

In any case, we are in luck. This project already has `str_to_bool`: https://github.com/elifesciences/data-hub-core-airflow-dags/blob/c4b0cda4a9096e167a370dce12742519c5f960a6/data_pipeline/utils/pipeline_config.py#L70

Although still no option of providing a default value directly. Perhaps it could be extended.